CODING_GUIDELINES.md

```diff
@@ -0,0 +1,50 @@
+# Mumble coding guidelines
```

Comment:
How did this end up in here?

Reply:
I tried to squash the commit but something went wrong. Im closing this pull request and making a new one up to date with master